dev/http.go

```diff
@@ -104,41 +104,6 @@ func (h *HTTPServer) findApp(name string) (*App, error) {
 	return app, nil
 }
 
-func (h *HTTPServer) hostForApp(name string) (string, string, error) {
```

Comment:
Unused method. I'm not sure what its original intent was. Given that many FQDNs can point to the same puma-dev app, I'm not sure why you'd need a canonical host per app. Perhaps @evanphx can weigh in?